### ISSUE TITLE
suposiciones de existencia las cartas

### ISSUE THREAD

**Issue by rodrigoogalde** [RodrigoToroIcarte/IIC2113-2023-1#123]
Hola!
En el enunciado no dice nada sobre si todas las cartas que están en los .txt existen en el .json, por lo que quería saber si podemos asumir esto.
Gracias!

**Comment by RodrigoToroIcarte** [comment]
Sí, las cartas que aparezcan en los .txt siempre existirán en el .json. No hay que chequear nada al respecto. Los títulos de las cartas estarán bien escritos. Además la primera línea siempre será la superestrella y en todos los test cases seremos consistentes con el formato.

También ver https://github.com/RodrigoToroIcarte/IIC2113-2023-1/issues/1

**Comment by rodrigoogalde** [comment]
Gracias profe!
Otra duda, para comprobar que una carta es igual a otra, esta debe ser igual en toda la información de la carta, o solo el título (u otros)

**Comment by RodrigoToroIcarte** [comment]
Solo el título. Si tienen el mismo título son copias de la misma carta.
